packages/components/package.json

```diff
@@ -35,7 +35,8 @@
     "@radix-ui/react-switch": "^1.0.1",
     "@radix-ui/react-toast": "^1.1.2",
     "ariakit": "^2.0.0-next.43",
-    "react-dropzone": "^14.2.3"
+    "react-dropzone": "^14.2.3",
+    "react-hook-form": "^7.43.2"
```

Comment:
I added this here because the other components are listed here but do we really need to be listing the package twice? I thought just having it in `peerDepdencies` would have been enough.

I'm planning on going with this for now to unblock my work since we can't quickly test these changes without publishing the packages. Then we can come back and test this a little more to see if this is actually needed later.